js/app.js

```diff
@@ -48,8 +53,13 @@ if ((doILikeAlpacas === 'yes') || (doILikeAlpacas === 'y')) {
   alert('Sorry to break it to you BUT wrong, I love alpacas!');
 
 }
+}
+favouriteAnimal();
```

Comment:
It's a good idea to put all your function calls in one spot. The standard is to have them all at the end of your document. It's very helpful for bigger projects and debugging!